src/Env.php

```diff
@@ -77,7 +77,7 @@ public static function getConfig($dir, $Label)
     {
         $Config = $dir . '/config/' . $Label . '.php';
         if (!file_exists($Config)) {
-            throw new \Exception($Config . ' not found in '.static::class.'::config()');
+            throw new \Exception($Config . ' not found in '.static::class.'::config(). Does this process have write permissions?');
```

Comment:
I feel like instead of writing that error message we should check with if(is_writable())